demo/avatar-advanced-demos.html

```diff
@@ -73,6 +73,32 @@ <h3>Localization</h3>
       </template>
     </vaadin-demo-snippet>
 
+    <h3>Screen reader announcements</h3>
+    <p>
+      When user joins or leaves the avatar group, a corresponding screen reader announcement is
+      dispatched using <code>iron-announce</code> custom event. In the example below, avatars change
+      very 3 seconds.
+    </p>
+    <vaadin-demo-snippet id="avatar-advanced-demos-sr" when-defined="vaadin-avatar-group">
+      <template preserve-content>
+        <vaadin-avatar-group></vaadin-avatar-group>
+        <script>
+          window.addDemoReadyListener('#avatar-advanced-demos-sr', function(document) {
+            const group = document.querySelector('vaadin-avatar-group');
+            group.items = [{name: 'Pekka Maanpää'}, {name: 'Jens Jansson'}];
+
+            var i = 0;
+
+            setInterval(function() {
+              const user = i % 2 === 0 ? {name: 'Serhii Kulykov'} : {name: 'Jens Jansson'};
+              group.splice('items', 1, 1, user);
+              i += 1;
+            }, 3000);
```

Comment:
I would probably increase the interval as sometimes it doesn't have enough time to pronounce the whole phrase. Moreover it overwhelms the channel with a lot of phrases that would come as a surprise for the user who is using screenreader and has joined the page 😅 
Probably makes sense to toggle the interval on `button` press and not from the beginning.

Reply:
Thanks, done.